expo/gbe/views.py

```diff
@@ -1417,8 +1431,11 @@ def review_volunteer(request, volunteer_id):
         return view_volunteer(request, volunteer_id)
     conference, old_bid = get_conf(volunteer)
     volunteer_prof = volunteer.profile
-    volform = VolunteerBidForm(instance=volunteer,
-                               prefix='The Volunteer')
+    volform = VolunteerBidForm(
+        instance=volunteer,
+        prefix='The Volunteer', 
+        available_windows=volunteer.conference.windows(),
+        unavailable_windows=volunteer.conference.windows())
```

Comment:
There's something broken here.  When I look at a volunteer bid that has no unavailable times selected, this displays as blank.  This works fine on edits, so there's something wrong here.

Test case:
http://localhost:8282/volunteer/review/462
if you use recent data from live or test.